app/views/curation_concerns/base/_form_files.html.erb

```diff
@@ -1,4 +1,27 @@
      <div id="fileupload">
+         <% if f.object.file_sets.present? > 0 %>
```

Comment:
`present?` returns a boolean. It doesn't make sense to compare it to an integer.